docs/pages/enroll-resources/machine-id/deployment/bitbucket.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Deploying Machine ID on Bitbucket Pipelines
+description: How to install and configure Machine ID on Bitbucket Pipelines
+---
+
+In this guide, you will configure Machine ID's agent, `tbot`, to run within a
+Bitbucket Pipelines workflow. The bot will be configured to use the `bitbucket`
+delegated joining method to eliminate the need for long-lived secrets.
+
+## How it works
+
+The `bitbucket` join method is a secure way for Machine ID bots to authenticate
+with the Teleport Auth Service without using any shared secrets. Instead, it
+makes use of an OpenID Connect token that Bitbucket Pipelines injects into the
```

Comment:
```suggestion
makes use of an OpenID Connect token that Bitbucket Pipelines inject into the
```

Reply:
Hmm, I think it's correct as written? "Bitbucket Pipelines" is a singular product so I think "injects" is correct here. If we said "Bitbucket pipelines" referring to individual workflows then we'd want the singular form, but I've tried to refer to them as "Bitbucket Pipelines workflows" or similar.

Amusingly, even Bitbucket frequently calls them workflows instead of pipelines.